provider/cluster/kube/k8s_integration_test.go

```diff
@@ -89,6 +92,39 @@ func TestNewClient(t *testing.T) {
 	assert.NoError(t, err)
 	assert.NotEqual(t, maxtries, tries)
 
+	logs, err := client.ServiceLogs(ctx, lid, svcname, true, nil)
+	require.NoError(t, err)
+	require.Equal(t, int(sstat.AvailableReplicas), len(logs))
+
+	log := make(chan string)
+	defer close(log)
```

Comment:
not really necessary, but it will cause a panic if `log` is written to after it's closed, which is nice for the tests